library/talaiot/build.gradle.kts

```diff
@@ -7,7 +7,7 @@ plugins {
 dependencies {
     api(project(":library:talaiot-logger"))
     api(project(":library:talaiot-request"))
-    implementation("guru.nidi:graphviz-java:0.8.3")
+    api("guru.nidi:graphviz-java:0.8.3")
```

Comment:
I've made this an `api` since there are still some parts of the `talaiot` library that depends on it.

Reply:
it makes sense